README.md

```diff
@@ -216,6 +217,44 @@ If you use the `HttpBodyStream` the whole transfer will be [chunked encoded](htt
 
 Check out the `examples` folder how your computation could look like.
 
+### Streaming requests
+
+Streaming requests makes it possible to send big amount of data in small chunks from the client to the server. 
+Every chunk will be sent directly to server, so there is no need to buffer the data on the client.
+
+The body of the request object in your callback and middleware function will be a `HttpBodyStream`, which can used as a standard [ReactPHP stream](https://github.com/reactphp/stream).
```

Comment:
Should be `ReadableStreamInterface`?

Also, what if any of the middlewares buffers this?